bin/tests/it/k8s.rs

```diff
@@ -1338,20 +1421,20 @@ async fn test_k8s_enrichment() {
         }
 
         // Wait for the data to be received by the mock ingester
-        tokio::time::sleep(tokio::time::Duration::from_millis(10_000)).await;
+        tokio::time::sleep(tokio::time::Duration::from_millis(3_000)).await;
```

Comment:
Could this lower time cause integration tests to be unreliable?

Reply:
no,, it is enough. legacy attempts to fight with flakiness of test. now we properly wait for pod state.